README.md

```diff
@@ -18,25 +18,18 @@
 ---
 
 ## Requirement
+
 - python 3.10
 - postgresql
 
 ## How to contribute?
-1. Fork or clone the project
-2. Create a [New Application](https://discord.com/developers/applications).
-3. Create a bot by going to Bot -> Add Bot -> Yes, do it!
-4. Copy the bot token and paste it into the `BOT_TOKEN` environment variable (see the next step).
-5. Rename `.env.example` to `.env` and edit `.env` to your bot.
-6. Create python virtual environment by using `python -m venv env`.
-7. Enter the virtual environment by using command `source env/Scripts/Activate` (Linux) or `env\Scripts\Activate.bat` (Windows)
-8. Make sure you have installed `poetry`. Install it using `pip install poetry` on your virtual environment.
-9. Install the depedencies using `poetry install`.
-10. Execute the database creation script from `bot\data\db.sql` on your database console.
-11. Set `BOT_DEBUG=1` to run the bot in debug mode on `.env`. Set to `0` only for production.
-12. To start the bot, use `poetry run task start`.
+
+Start to contribute to our community by following this [contributing guidelines](https://github.com/Iqrar99/WarnetBot/blob/main/CONTRIBUTING.md).
```

Comment:
```suggestion
Start contributing to our community by following the [contributing guideline](https://github.com/Iqrar99/WarnetBot/blob/main/CONTRIBUTING.md).
```

Also you can change the link into a relative path so we don't have to hardcode the link anymore.